src/components/comments/Comment.jsx

```diff
@@ -12,7 +17,40 @@ class Comment extends Component {
         <br />
         <Card bg="light">
           <Card.Header>
-            {this.props.author} - {this.props.created}
+            {this.props.author} - {this.props.created ?? "just now"}
+            <span>
+              <i
+                className={`icon icon-downvote ${
+                  this.state.downvote ? "voted" : null
+                }`}
+                onClick={
+                  !this.state.downvote &&
```

Comment:
This should be `(!this.state.downvote && !this.state.upvote)` because we're not allowing users to vote if they already voted. Even if, this doesn't work when I upvote and then downvote, for example: original score 3, I upvote to 4, and then downvote back to 3, although it should have been 2, since I "changed my mind". But I think we want to go with disabling voting if already voted in any case.

Reply:
I think I have fixed that. The voting count now displays the expected number according to the action.

Reply:
And why not just disabling voting after having voted? Because that's one of our requirements. And also, once we connect the back-end, it will be much easier (we will not have to worry about updating votes, etc.. just one request per vote), I worry this adds extra unnecessary complexity